cocotb/share/lib/embed/gpi_embed.c

```diff
@@ -51,6 +54,68 @@ static char *argv[] = { progname };
 
 static PyObject *pEventFn = NULL;
 
+
+/**
+ * @name    handle_venv
+ * @brief   Set 
+ * @ingroup python_c_api
+ *
+ * GILState before calling: N/A
+ *
+ * GILState after calling: N/A
+ *
+ * Checks if running in Python virtualenv and if so,
+ * calls Py_SetProgramName with correct path to interpreter
+ */
+
+void handle_venv(void)
```

Comment:
- Mark this function `static`, it's private to this class.
- If you use a more descriptive function name like `venv_set_program_name()` or `set_program_name_in_venv()` you can avoid the rather long and not overly helpful Doxygen comment.